skipruntime-ts/server/src/rest.ts

```diff
@@ -77,7 +77,7 @@ export function createRESTServer(runtime: SkipRuntime): express.Express {
       res.status(200).json({});
     } catch (e: unknown) {
       if (e instanceof UnknownCollectionError) {
-        res.status(400).json("Bad request");
+        res.status(404).json("Not Found");
```

Comment:
We should probably omit the body as a `404` response status implies the "Not found" information.